packages/RichTextEditing-Tests.package/RichTextStructureEditorTests.class/instance/testUS42FontCanBeReset.st

```diff
@@ -7,7 +7,7 @@ testUS42FontCanBeReset
 		addFontStructureToText;
 		setFont: self fontAtlanta22;
 		deny: (self textHasFont: self structureEditor defaultFont).
```

Comment:
Tow tests?

Reply:
Not useful here, as it tests before and after behavior to check if both is correct.